app/src/main/java/com/google/samples/apps/nowinandroid/ui/NiaApp.kt

```diff
@@ -196,14 +196,15 @@ fun NiaApp(
                                 onNavigationClick = { appState.navigateToSearch() },
                             )
                         }
-
-                        NiaNavHost(appState = appState, onShowSnackbar = { message, action ->
-                            snackbarHostState.showSnackbar(
-                                message = message,
-                                actionLabel = action,
-                                duration = Short,
-                            ) == ActionPerformed
-                        })
+                        NiaAnimation {
```

Comment:
It's not quite clear how adding an animation on the snackbar solves an issue on the topAppBar. Can you please adda gif or a video showing your fix?

Reply:
ACTUALLY THE GLITCH HERE IS AN INSTANT CHANGE OF PICTURE SO THE HUMAN EYE CATCH THIS DISCONTINUITY ADDING ANIMATION WILL MAKE THE PICTURE MOVING SMOOTHLY FOR THE HUMAN EYE AND MAKE IT CONTINUES
HERE IS A VIDEO TO SHOW THE NEW SITUATION
[device-2023-08-05-102456.webm](https://github.com/android/nowinandroid/assets/1716129/f501983a-f109-4bbb-86d8-bf3f17019a7d)

Reply:
@tunjid any thing we shall take into consideration here?